lib/tasks/evm.rake

```diff
@@ -96,7 +96,13 @@ namespace :evm do
       "redeployment"   => 5,
       "upgrade"        => 6
     }
+
     status = EvmApplication.deployment_status
+    if status == "new_replica"
+      puts "Database contains the following server records:"
+      MiqServer.all.each { |server| puts "  #{server.id} : #{server.guid}" }
```

Comment:
```suggestion
      MiqServer.all.each { |server| puts "  #{server.guid}: id: #{server.id}, name: #{server.name}, zone: #{server.zone.description}" }
```

Other possible server fields - `status`, `started_on`, `stopped_on`, `last_heartbeat`, `region_number` (though that's evident via the id)

Reply:
Yeah, let's start with this and see what else we need to add in the future